iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/ReplaceThreeWindingsTransformersBy3TwoWindingsTransformersTest.java

```diff
@@ -0,0 +1,318 @@
+                   Extension [unknownExtension1, unknownExtension2] of threeWindingsTransformer 3WT will be lost
+                """, TestUtil.normalizeLineSeparator(sw1.toString()));
+    }
+
+    @Test
+    void replaceFlowsTest() {
+        modifyNetworkForFlowsTest();
+
+        ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers replace = new ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers();
+        replace.apply(network);
+
+        TwoWindingsTransformer t2w1 = network.getTwoWindingsTransformer("3WT-Leg1");
+        TwoWindingsTransformer t2w2 = network.getTwoWindingsTransformer("3WT-Leg2");
+        TwoWindingsTransformer t2w3 = network.getTwoWindingsTransformer("3WT-Leg3");
+
+        TwtData t3wData = new TwtData(t3w, 0.0, false);
+        setStarBusVoltage(t3wData, t2w1.getTerminal2().getBusView().getBus());
+
+        BranchData t2w1Data = new BranchData(t2w1, 0.0, false, false);
+        BranchData t2w2Data = new BranchData(t2w2, 0.0, false, false);
+        BranchData t2w3Data = new BranchData(t2w3, 0.0, false, false);
+
+        double tol = 0.000001;
+        assertEquals(t3wData.getComputedP(ThreeSides.ONE), t2w1Data.getComputedP1(), tol);
+        assertEquals(t3wData.getComputedP(ThreeSides.TWO), t2w2Data.getComputedP1(), tol);
+        assertEquals(t3wData.getComputedP(ThreeSides.THREE), t2w3Data.getComputedP1(), tol);
+    }
+
+    private void modifyNetworkForFlowsTest() {
+        addVoltages(t3w.getLeg1().getTerminal().getBusView().getBus(), t3w.getLeg2().getTerminal().getBusView().getBus(), t3w.getLeg3().getTerminal().getBusView().getBus());
+        addVoltages(network.getThreeWindingsTransformer(t3w.getId()).getLeg1().getTerminal().getBusView().getBus(),
+                network.getThreeWindingsTransformer(t3w.getId()).getLeg2().getTerminal().getBusView().getBus(),
+                network.getThreeWindingsTransformer(t3w.getId()).getLeg3().getTerminal().getBusView().getBus());
+    }
+
+    @Test
+    void replaceNodeBreakerTest() {
+        modifyNetworkForNodeBreakerTest();
+
+        assertEquals(3, network.getVoltageLevelCount());
+        assertEquals(1, network.getThreeWindingsTransformerCount());
+        assertEquals(0, network.getTwoWindingsTransformerCount());
+
+        ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers replace = new ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers();
+        replace.apply(network);
+
+        assertEquals(4, network.getVoltageLevelCount());
+        assertEquals(0, network.getThreeWindingsTransformerCount());
+        assertEquals(3, network.getTwoWindingsTransformerCount());
+    }
+
+    private void modifyNetworkForNodeBreakerTest() {
+        Network expectedNetwork = createThreeWindingsTransformerNodeBreakerNetwork();
+        t3w = expectedNetwork.getThreeWindingsTransformer("3WT");
+        network = createThreeWindingsTransformerNodeBreakerNetwork();
+    }
+
+    @Test
+    void testApplyChecks() {
```

Comment:
Consider for more consistency to name the method ```applyChecksTest``` instead

Reply:
Done